roles/setup/templates/systemd_receptor.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Ansible Receptor for Linux
+After=network.target
+StartLimitIntervalSec=0[Service]
+
+[Service]
+ExecStart={{ receptor_install_dir }}/receptor --config {{ receptor_config_path }}/receptor.conf
+TimeoutStopSec=5
+Type=simple
+Restart=always
+RestartSec=1

```

Comment:
can we add this under [Service]

StandardOutput=append:/var/log/receptor/receptor.log